docs/oid-info.md

```diff
@@ -23,35 +23,49 @@ The `.1` is added to the root OID for sigstore for all OIDs set by Fulcio.
 This contains the `issuer` claim from the OIDC Identity Token that was
 presented at the time the code signing certificate was requested to be created.
 This claim is the URI of the OIDC Identity Provider that digitally signed the
-identity token.
+identity token. For example: https://token.actions.githubusercontent.com.
 
-### 1.3.6.1.4.1.57264.1.2 | GitHub Workflow Trigger
+### 1.3.6.1.4.1.57264.3.1 | Source URI
 
-This contains the `event_name` claim from the GitHub OIDC Identity token that
-contains the name of the event that triggered the workflow run.
-[(docs)][github-oidc-doc]
+Should include a fully qualified source repository URL that the build was based
+on.
 
-### 1.3.6.1.4.1.57264.1.3 | GitHub Workflow SHA
+### 1.3.6.1.4.1.57264.3.2 | Source Digest
 
-This contains the `sha` claim from the GitHub OIDC Identity token that contains
-the commit SHA that the workflow run was based upon. [(docs)][github-oidc-doc]
+An immutable reference to a specific version of the source code that the build
+was based upon. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
 
-### 1.3.6.1.4.1.57264.1.4 | GitHub Workflow Name
+### 1.3.6.1.4.1.57264.3.3 | Source Ref
 
-This contains the `workflow` claim from the GitHub OIDC Identity token that
-contains the name of the executed workflow. [(docs)][github-oidc-doc]
+The source ref that the build run was based upon. For example: `refs/head/main`.
 
-### 1.3.6.1.4.1.57264.1.5 | GitHub Workflow Repository
+### 1.3.6.1.4.1.57264.3.4 | Build Config URI
 
-This contains the `repository` claim from the GitHub OIDC Identity token that
-contains the repository that the workflow run was based upon.
-[(docs)][github-oidc-doc]
+A reference to the initiating build instructions.
 
-### 1.3.6.1.4.1.57264.1.6 | GitHub Workflow Ref
+### 1.3.6.1.4.1.57264.3.5 | Build Config Digest
 
-This contains the `ref` claim from the GitHub OIDC Identity token that contains
-the git ref that the workflow run was based upon.
-[(docs)][github-oidc-doc]
+An immutable reference to the specific version of the top-level build
+instructions. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
+
+### 1.3.6.1.4.1.57264.3.6 | Build Siger URI
+
+Reference to specific build instructions that are responsible for signing. Can be the same as Build Config URI. For example a reusable workflow in GitHub Actions or a Circle CI Orbs.
+
+
+### 1.3.6.1.4.1.57264.3.7 | Build Siger Digest
+
+An immutable reference to the specific version of the build instructions that is responsible for signing. Should include the digest type followed by the digest, e.g. `sha1:abc123`.
+
+
+### 1.3.6.1.4.1.57264.3.8 | Build Trigger
+
+The event or action that triggered the build.
+### 1.3.6.1.4.1.57264.3.9 | Runner Environment
```

Comment:
`platform-hosted` environments are always ephemeral. In case of `self-hosted` environment they are most likely not ephemeral but can be. Should there be a some optional clarifications like version of environment for `platform-hosted` or host name for `self-hosted`?

Reply:
Good point, we could construct this as a user-agent string that would support versions and extra metadata. 

I'm anticipating that each CI provider probably ends up having custom values that make most sense for them and would be hard to standardize across providers. In GitHub's case, it might be clearer to say `runner_env: github` instead of `runner_env: platform-hosted`.